sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -282,3 +279,14 @@ private[hive] class SparkExecuteStatementOperation(
     }
   }
 }
+
+object SparkExecuteStatementOperation extends Logging {
+  def getTableSchema(output: Seq[Attribute]): TableSchema = {
```

Comment:
Oh, I see. Sorry, I just thought it's an example to remove `SparkSession` stuff. I'll fix that again. Thank you for fast reply.